start-docker.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+docker-compose run --service-ports zotonic sh
```

Comment:
Not really useful IMHO, as this command is already shown in the documentation.

Reply:
I added it so that it is easier to discover without reading the docs.
I continuously forget the command, and in this way I can easily use it.